anthologize/README.md

```diff
@@ -188,19 +190,23 @@ where `<x>` is the one-letter code of the conference (e.g., `N` for NAACL) and `
 
 ## 4. Package and send the Anthology directory
 
-Package up the anthology directory and send it to your book chair.
-If you are the book chair, please contact the Anthology Director.
+Package up the anthology directory:
 
     tar czvf <conference>_anthology.tgz anthology
 
+Send the resulting file (`<conference>_anthology.tgz`), or a link to it, by email to the Anthology Director.
```

Comment:
Is this right?

Reply:
"Email a link to the file to the Anthology Director (please do not send the file as an email attachment)".